gql/resolver.go

```diff
@@ -601,8 +601,15 @@ func (dr *dealResolver) sealingState(ctx context.Context) string {
 		log.Warnw("error getting sealing status for sector", "sector", dr.SectorID, "error", err)
 		return "Sealer: Sealing"
 	}
-
-	return "Sealer: " + string(si.State)
+	if len(si.Deals) > 0 {
```

Comment:
I don't think you actually need the if statement.
I believe `for _, d := range si.Deals` will just run zero times if si.Deals is nil